crates/sui-core/src/jsonrpc_index.rs

```diff
@@ -1558,40 +1533,29 @@ impl IndexStore {
     /// `get_Balance()` queries. Notice that db read is performed with `spawn_blocking` as that is
     /// expected to block
     #[instrument(skip(self))]
-    pub async fn get_all_balance(
+    pub fn get_all_balance(
```

Comment:
its possible that these changes will impact the jsonrpc server (depending on if any of the async stuff was actually async), but its unlikely since today we were just blocking the task doing db lookups.